### PR TITLE
 change permission on organization level

### PR DESCRIPTION
**Pull request by vishalajackus**
## Purpose
Change permission for organization type to organization level

## Approach
Add in permission class

### Further Info
Ticket number: #000
 
@mentions of the person or team responsible for reviewing proposed changes.

### CODE REVIEW

File: core/views/organization.py [hunk 1]
```diff
@@ -98,6 +98,6 @@ class OrganizationTypeViewSet(viewsets.ModelViewSet):
 
     filter_fields = ('name',)
     filter_backends = (DjangoFilterBackend,)
-    permission_classes = (IsSuperUser,)
+    permission_classes = (IsOrgMember,)
```
Comment (RadhikaPPatel) [inline core/views/organization.py:101]: Needs to be AllowOnlyOrgAdmin

**Review by RadhikaPPatel — CHANGES_REQUESTED** [review]
Correct permission group

File: core/views/organization.py (hunk 1 above)
Comment (vishalajackus) [reply core/views/organization.py]: Added

**Review by RadhikaPPatel — APPROVED** [review]
Looks good

**Pull request merged**